### DISCUSSION TITLE
How to use the system time parameter to control the synchronization scope of blob data?

### DISCUSSION THREAD

**Discussion by sin90kk** [airbytehq/airbyte#34593] [Connector Questions]
We plan to use a blob connector to integrate blob data into Big Query.
 All files in the blob are stored in the same folder(can not be changed), and new files are generated every day.
How can the system time parameter be utilized to control the scope of data integration? 
For instance, on January 29, 2024, only integrate data with names containing '20240129,' and on January 30, 2024, only integrate data with names containing '20240130.' 
Can the current connector automate the data integration based on these criteria?

**Comment by sajarin** [comment]
I don't know if our current connectors can sync based on the name but, you can set the sync frequency to be everyday? Would that not work for your use case?

[a comment by an automated account is not shown]
